lib/Config/LegacyConfig.php

```diff
@@ -54,7 +54,12 @@ public function __construct(
         $this->updateInterval = 3600;
     }
 
-    public function read($configPath, $createIfNotExists = false)
+    /**
+     * @param false $createIfNotExists
+     *
+     * @return void
+     */
+    public function read($configPath, bool $createIfNotExists = false)
```

Comment:
Shouldn't this `@param` be `bool`? Otherwise could it maybe be removed, since it's not even used.

Reply:
I guess psalm says it should be false, let's remove unsued vars later this is just the easy fixes